db/queries.py

```diff
@@ -3,44 +3,82 @@
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(
+        Message.objects.filter(text__icontains=word)
+    )
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return list(
+        Chat.objects.filter(title__startswith="Untitled")
+    )
 
 
 def get_users_who_sent_messages_in_2015() -> list[str]:
```

Comment:
don't return type equals to list(tuple(str))?
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#django.db.models.query.QuerySet.values_list